src/list.c

```diff
@@ -299,3 +299,17 @@ struct list *list_deep_clone_strs(struct list *list)
 
 	return clone;
 }
+
+int list_longer_than(struct list *list, int count)
+{
+	struct list *item;
+
+	item = list_head(list);
```

Comment:
I think you can use `list` itself as you did with count since you are not doing any modification in the list content for this.

Reply:
Common, that is a 10 line function. Compared with other stuff in this code base, this is way far from hurting the maintenance of this code.